Inspicio/Inspicio.csproj

```diff
@@ -46,6 +46,7 @@
     <Folder Include="Data\Migrations\" />
     <Folder Include="Migrations\" />
     <Folder Include="Migrations\" />
+    <Folder Include="Migrations\" />
```

Comment:
Did a merge go wrong here? This folder shouldn't be mentioned multiple times in the csproj file.